homeassistant/components/wyoming/__init__.py

```diff
@@ -23,6 +23,7 @@
     Platform.SELECT,
     Platform.SWITCH,
     Platform.NUMBER,
+    Platform.CONVERSATION,
```

Comment:
This is not a satellite platform. Instead, it should only be handled by `data.py`.
```suggestion
```